doc/articles/migrating-to-uno-sdk.md

```diff
@@ -0,0 +1,195 @@
+---
+uid: Uno.Development.MigratingToUnoSdk
+---
+# Migrating to Uno.Sdk
+
+The Uno.Sdk provides several enhancements to further simplify Uno Projects. It is important to remember that the Uno.Sdk is designed to reduce the noise that you need in your projects, while still ensuring that you have the ability to override values that come from the Sdk itself. This means that values such as the `SupportedOSPlatformVersion` which have been in the template's `Directory.Build.props` in the past, now have a default value set to the minimum supported by Uno Platform. If you need to be more restrictive for your specific project you can provide your overrides for the platform(s) that you need.
+
+## Create a global.json
+
+It is recommended that you create a global.json in your solution root directory.
+
+```json
+{
+  "msbuild-sdks": {
+    "Uno.Sdk": "{Current Version of Uno.WinUI/Uno.Sdk}",
+    "Microsoft.Build.NoTargets": "3.7.56"
+  }
+}
+```
+
+> [!NOTE]
+> The Microsoft.Build.NoTargets Sdk has been used for the last several versions in the Uno Templates. Depending on which version of the templates this may have been called `{YourProject}.Base` or `{YourProject}.Shared`. With the introduction of the global.json you may want to update the Sdk in the csproj to remove the version and use the version provided by the global.json instead. This makes it easier to manage your Sdk versions centrally.
```

Comment:
```suggestion
> The Microsoft.Build.NoTargets Sdk has been used for the last several versions in the Uno Templates. Depending on which version of the templates this may have been called `{YourProject}.Base` or `{YourProject}.Shared`. With the introduction of the `global.json` file, you may want to update the `Sdk` in the csproj to remove the version and use the version provided by the global.json instead. This makes centralizes your Sdk versions.
```